openkfw/TruBudget
# frontend: Call authproxy signout on logout

### Checklist




### Description


### How to test

1. If testing locally, start an Auth Proxy instance
3. Start TB configured to use Auth Proxy (AUTHPROXY_ENABLED, AUTHPROXY_JWS_SIGNATURE)
2. Login with Auth Proxy
3. Logout 
4. You should be logged out of TB, and another window should open to sign out from Auth Provider
5. If logging in normally, no additional window should pop up




Closes #1855
